js/ui/file_manager/ui.file_manager.js

```diff
@@ -363,6 +372,16 @@ class FileManager extends Widget {
             */
             customizeDetailColumns: null,
 
+            /**
+            * @name dxFileManagerOptions.onCurrentDirectoryChanged
+            * @extends Action
+            * @type function(e)
+            * @type_function_param1 e:object
```

Comment:
Any object? Can we be more specific?

Reply:
The event needs to update something on a web page when current directory was changed. All required info, you can get from file manager's instance, such as current directory path.